src/mapml/handlers/ContextMenu.js

```diff
@@ -723,6 +770,85 @@ export var ContextMenu = L.Handler.extend({
         L.DomEvent.stop(e);
         this._focusOnLayerControl();
       }
+    } else if (key === 38) { //up arrow
+      if (!this._coordMenu.hasAttribute('hidden')&&document.activeElement.shadowRoot.activeElement.innerHTML === this._coordMenu.children[0].innerHTML) {
```

Comment:
Better to continue this on two or more lines than remove spaces to fit it all.  Good job finding something to compare it to, but maybe add what you're looking for as a trailing comment e.g. `// map` etc.